pkg/meta/repodb/repodb.go

```diff
@@ -123,7 +128,18 @@ type RepoDB interface { //nolint:interfacebloat
 	// ToggleBookmarkRepo adds/removes bookmarks on repos
 	ToggleBookmarkRepo(ctx context.Context, reponame string) (ToggleState, error)
 
-	PatchDB() error
+	// UserDB profile/api key CRUD
+	GetUserProfile(ctx context.Context) (UserProfile, error)
```

Comment:
I think we should stick with 1 name for the methods and for the bucket names. The bucket is currently called `UserData` and the method `GetUserProfile`.